.github/workflows/cicd.yml

```
@@ -0,0 +1,59 @@
+            ECR_REGISTRY: ${{ secrets.ECR_REGISTRY }}
+            ECR_REPOSITORY: ${{ secrets.ECR_REPOSITORY }}
+            IMAGE_TAG: ${{ secrets.IMAGE_TAG }}
+
+        steps:
+            -   name: Checkout code
+                uses: actions/checkout@v4
+
+            -   name: Check ktlint format
+                run: ./gradlew ktlintCheck
+
+            -   name: Build gradlew
```

YAMLlint (1.37.1)

[error] too many spaces after hyphen

(hyphens)